pages/index.tsx

```diff
@@ -1,5 +1,7 @@
 import { GetServerSideProps } from 'next';
-import { makeStyles, createStyles, Typography, Theme, List, Grid, Button } from '@material-ui/core';
+import { Typography, Theme, List, Grid, Button } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
```

Comment:
Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```